client/components/common/tce-core/AddElement/SelectElement.vue

```diff
@@ -56,20 +57,29 @@ export default {
       return Math.min(this.elements.length, this.rowSize);
     },
     elements() {
-      const result = filter(this.registry, it => it.type !== 'ASSESSMENT');
-      if (this.assessments.length) {
+      const result = filter(this.registry, it => !isQuestion(it.type));
+      if (this.questions.length) {
         result.push({
```

Comment:
Should we check individual subtypes (`ASSESSMENT`, `REFLECTION`) before we insert them?